ajbool--0.0.1--1.0.0.sql

```diff
@@ -0,0 +1,20 @@
+
+DO $$
+BEGIN
+ EXECUTE $E$ ALTER FUNCTION ajbool_in(cstring) PARALLEL SAFE $E$;
```

Comment:
EXECUTE is necessary here because we want to catch a syntax error when we get here, not when the DO block is  parsed

Reply:
it's a nice idea; one must not forget to add each new function then, but still better than the other approach that we have right now.